content/management/configuration-api/index.mdx

```diff
@@ -687,6 +687,15 @@ Here's what you need to know about **webhooks**:
 - They can be generated by both Web and RTM API actions. 
 - To receive webhooks, you need to [register](#register-webhook) them first.
 
+<Warning>
+
+Note: When your server receives a webhook from LiveChat, it should respond with **HTTP 200** response. Otherwise, LiveChat will retry sending the webhook to your service for a number of times unless it receives the correct **HTTP 200** response.
+
+The timeout is set to ~10 seconds and if in that time we won't receive **HTTP 200** response we'll retry sending the webhook up to 10 times within 6 hours.
+
+</Warning>
+
```

Comment:
Should the note about suggested way of working with webhooks be also included here?

Reply:
The text looks really long then, and I don't think it's that important. Developer should figure it out by themselves, I guess.
We added this note to the Guide document, that have tips for such situations.